sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/config/TransportConfig.java

```diff
@@ -92,11 +95,17 @@ public static Long getHeartbeatIntervalMs() {
                 continue;
             }
             ipPortStr = ipPortStr.trim();
-            if (ipPortStr.startsWith("http://")) {
-                ipPortStr = ipPortStr.substring(7);
-            }
-            int index = ipPortStr.indexOf(":");
-            int port = 80;
+			int port = 80;
```

Comment:
It's strange, it looks good in my IDE.

Reply:
> It's strange, it looks good in my IDE. Let me check.

Yes, it depends on your editor's preference settings.
Maybe you have the setting using TAB instead of SPACE and setting the width of the TAB to 4 spaces' width.
Sentinel and some other projects use the style of plain spaces with an indent of 4 spaces.

Reply:
> > It's strange, it looks good in my IDE. Let me check.
> 
> Yes, it depends on your editor's preference settings.
> Maybe you have the setting using TAB instead of SPACE and setting the width of the TAB to 4 spaces' width.
> Sentinel and some other projects use the style of plain spaces with an indent of 4 spaces.

Great!